helm/vertical-pod-autoscaler-app/templates/crd-patch/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: annotate-job-sa
```

Comment:
Can you add the vpa prefix here? Otherwise this looks nice

Reply:
Done.